### PR TITLE
Update catalan messages.po

### PR DESCRIPTION
**Pull request by ivanbea**
New lines added, new lines localized.

Check it please @jordimas @darccio @surfdude29

### CODE REVIEW

File: src/locale/locales/ca/messages.po [hunk 1]
```diff
@@ -3173,7 +3173,7 @@ msgstr "Llistes"
 
 #: src/components/dms/BlockedByListDialog.tsx:39
 msgid "Lists blocking this user:"
-msgstr ""
+msgstr "Llistes que bloquegen aquest usuari"
```
Comment (surfdude29) [inline src/locale/locales/ca/messages.po:3176]: There's a colon missing at the end

File: src/locale/locales/ca/messages.po [hunk 2]
```diff
@@ -584,15 +584,15 @@ msgstr "Confirmes que vols eliminar la contrasenya de l'aplicació \"{name}\"?"
 
 #: src/components/dms/MessageMenu.tsx:124
 msgid "Are you sure you want to delete this message? The message will be deleted for you, but not for the other participant."
-msgstr ""
+msgstr "Estàs segur que vols esborrar aquest missatge? El missatge s'esborrarà per a tu, però no per als altres participants."
```
[the remark this answers is not in the document]
Comment (ivanbea) [reply src/locale/locales/ca/messages.po:587]: Thank you again for your corrections, and also to learn Catalan by logic :)

[the remark this answers is not in the document]
Comment (surfdude29) [reply src/locale/locales/ca/messages.po:587]: Hehe yeah I certainly am picking up some :)

[the remark this answers is not in the document]
Comment (surfdude29) [reply src/locale/locales/ca/messages.po:587]: Btw I don't know if you already saw @jordimas's two corrections here?

https://github.com/ivanbea/social-app/commit/441cb9ba4be4f5dc5e53a37af653495b0bf7846b#r142228824

[the remark this answers is not in the document]
Comment (ivanbea) [reply src/locale/locales/ca/messages.po:587]: Thanks again, I have not spotted them

**Review by pfrazee — APPROVED** [review]
Thanks!!

**Pull request merged**